digdag-standards/src/main/java/io/digdag/standards/command/kubernetes/KubernetesClientConfig.java

```diff
@@ -19,7 +19,7 @@ public static KubernetesClientConfig create(final Optional<String> name,
             final Config systemConfig,
             final Config requestConfig)
     {
-        if (requestConfig.has("kubernetes")) {
+        if (requestConfig != null && requestConfig.has("kubernetes")) {
```

Comment:
This can cause problems.
For example, this may be the case of some configuration is added to `Kubernetes CommandExecutor` in the future.
Kubernetes CommandExecutor`s settings such as pod are thought to be configured from requestConfig as same as Docker CommandExecutor.
Under the conditions of this line, if you read the kubernetes authentication information from systemConfig and try to set up resources such as pod from requestConfig, it will try to read the authentication information from requestConfig, so the expected operation will not be performed
```yaml
_export:
  kubernetes:
     pod: something_configure
```

Reply:
@TrsNium Let me confirm your point. In my understand, you describe the risk on not modification by PR  but the original code itself. Is this right ? Then you want to merge system config and request config.

Reply:
@yoyama 
> you describe the risk on not modification by PR but the original code itself. Is this right ?Then you want to merge system config and request config.
That's right.

Reply:
@TrsNium @yoyama 
Thank you reviewing.
RequestConfig is merged based on SystetmConfig.
Please review again.